src/components/layout/nav/signup/SignUp.jsx

```diff
@@ -0,0 +1,75 @@
+import React, { useRef } from "react";
+import Button from "react-bootstrap/Button";
+import Col from "react-bootstrap/Col";
+import Row from "react-bootstrap/Row";
+import Form from "react-bootstrap/Form";
+import "./SignUp.scss";
+import { NavLink } from "react-router-dom";
+import useDeviceSizes from "../../useDeviceSizes";
+
+const SignUp = () => {
+  const passwordOfUser = useRef(false);
+  const emailOfUser = useRef(false);
+  const { widthOfDevice } = useDeviceSizes();
+  const breakPointsToSmallDevices = 765;
+
+  //This function will send the input values to check auth with 'emailOfUser.current.value' and 'passwordOfUser.current.value';
+  const handleLoginInputs = () => null;
+
+  return (
+    <>
+      <Col
+        xl={11}
+        className={`signupBox ${
+          widthOfDevice < breakPointsToSmallDevices ? "signUpBoxResponsive" : ""
```

Comment:
why do you need this instead of a media query? i'm really confused

Reply:
You are definitely right, I was thinking to stick to this method but after giving up on it for every components, I shouldn't have use it.